apollo-training/src/server.js

```diff
@@ -24,11 +24,11 @@ export default class Server {
    * @returns -Instance of Current Object
    */
   bootstrap() {
-    this._initHelmet();
-    this._initCompress();
-    this._initCors();
-    this._initJsonParser();
-    this._initMethodOverride();
+    this.initHelmet();
+    this.initCompress();
+    this.initCors();
+    this.initJsonParser();
+    this.initMethodOverride();
 
```

Comment:
Why are you using them?

Reply:
we are using and initialized some old packages, that'a why we are using this.